main.py

```diff
@@ -88,7 +88,28 @@ async def on_message(message: Message):
             await message.channel.send("You need to use !list first, sugar.")
             return
 
-        await command_play(message)
+        command_args = message.content.split()[1:]
+        skip_count = 0
+        if len(command_args) > 1:
+            bust_index = 0
+            try:
+                # Expects a positive integer
+                bust_index = int(command_args[1])
```

Comment:
```suggestion
                bust_index = int(command_args[0])
```

Now that `command_args` consists of only arguments.